src/styles/index.scss

```diff
@@ -1,3 +1,64 @@
+@import './variables.scss';
```

Comment:
Ensure that the `variables.scss` file contains all the variables used in this file, such as `$calendar__day-gap`, `$calendar__day-dimension`, `$calendar__padding`, `$calendar__border-color`, `$calendar__background-color`, `$calendar__day-hover-color`, `$calendar__transition-duration`, `$font-size-day-number`, and `$font-family-day-number`.